Grow.c

```diff
@@ -3761,7 +3764,8 @@ int reshape_container(char *container, char *devname,
 	 */
 	ping_monitor(container);
 
-	if (!forked && !c->freeze_reshape)
```

Comment:
@mtkaczyk The check was added in 40b941b813e22b33ff9c70831415f5b65e0fd70b

It states:

> Grow: Do not fork via systemd if freeze_reshape is set
> 
> Mdadm should not run 'grow-continue' unit file for container if
> '--freeze-reshape' argument is passed. Otherwise it will be ignored,
> and reshape will start anyway.

Reply:
Looks like this check might not be needed rn if we rely on just being in initrd. A helper function now should be easy to implement. Let me know what do you think bout it.

Reply:
Yes, LGTM.